solutions/capacity-scenario/readme.md

```diff
@@ -0,0 +1,217 @@
+
+![](Images/8.png)
+
+10. Provide a description and add a secret
+
+![](Images/9.png)
+
+11. Copy the value of this new secret and save it securely before navigating away from this page
+
+![](Images/10.png)
+
+12. Use this link to initiate the setup of the pre-requisites. Use the appid and secret created in the 
+previous steps. Custom deployment - Microsoft Azure [here](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoftgraph%2Fdataconnect-solutions%2Fmain%2FARMTemplates%2FMGDC%20Extraction%20Pre%20Reqs%2Fazuredeploy.json?token=AATN3TJ6UQWU7TFMZ2R6ZW3ASL5JQ)
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoftgraph%2Fdataconnect-solutions%2Fmain%2FARMTemplates%2FMGDC%20Extraction%20Pre%20Reqs%2Fazuredeploy.json?token=AATN3TJ6UQWU7TFMZ2R6ZW3ASL5JQ"><img src="https://camo.githubusercontent.com/bad3d579584bd4996af60a96735a0fdcb9f402933c139cc6c4c4a4577576411f/68747470733a2f2f616b612e6d732f6465706c6f79746f617a757265627574746f6e" alt="Deploy Environment in Azure" /></a>
+
+The link above sets up the pre-requisites to using the information oversharing template, which are:
+
+- Create a Synapse Workspace
+- Create a Spark Pool for the Synapse workspace
+- Create a storage account for the extracted data
+- Grant permission to the Synapse workspace & the MGDC Service Principal to the storage account as Blob Data Contributor
+- Create an Azure SQL Server
+- Create a sample database within the Azure SQL Server.
+
+By clicking on the above button (or navigating to the linked URL), users will be brought to the Azure portal on the Custom deployment page.
+
+On that screen, on top of providing information about the resource group and region to deploy the components into, they will need to provide the following information:
+
+- Application Id to be used by MGDC (from step #3, ending in 9826)
+- Application secret for that app
+- A new password for the Azure SQL Server
+
+Once all required information has been provided, click on the Review + create button at the bottom of the page:
+
+
+![](Images/11.png)
+
+This will validate that the information provided to the template is correct. Once the information has been validated, click on the Create button at the bottom of the page.
+
+![](Images/12.png)
+
+This will initiate the deployment. It should normally take about 5 minutes for the whole deployment to 
+complete.
+
+## Synapse pipeline template
+
+1.  After the pre-reqs are complete, navigate to the Synapse workspace just created
+
+![](Images/21.png)
+
+2.  Open the Synapse Studio
+
+![](Images/22.png)
+
+1.  Navigate to "Integrate -> Add new resource -> **Browse gallery**"
+
+![](Images/23.png)
+
+4.  Search for "SharePoint" and select the "Unlock Capacity Analytics and Insights for SharePoint and OneDrive using Microsoft 365 Datasets" template and Continue
+
+![](Images/24.png)
+
```

Comment:
I assume the template will not show here yet.
If so, we should have a note: "Note: You won't see the capacity template here until it becomes available."